bundles/best_practices.rst

```diff
@@ -75,7 +76,7 @@ configuration options (see below for some usage examples).
 Directory Structure
 -------------------
 
-The basic directory structure of an AcmeBlogBundle must read as follows:
+The basic directory structure of an ``AcmeBlogBundle`` must read as follows:
```

Comment:
I am against this change. We usually do not enclose bundle names with backticks.